### PR TITLE
Eventrouter image upgrade

### PR DESCRIPTION
**Pull request by pajaxsg**

### CODE REVIEW

File: .bravo/config.yaml [hunk 1]
```diff
@@ -12,28 +12,16 @@ pipeline:
...
-        chartPath: charts/eventrouter
-        lint: true
 
     "Build Docker Image":
       imageBuild: &imageBuild
         push: false
         name: platform/eventrouter
         tags:
-          - 0.3.0.pes1.0
+          - 0.3.0.pes1.1
```
Comment (dakamarko) [inline .bravo/config.yaml:21]: Usually we only increase first number since we do not need more granular versioning here. `0.3.0.pes2`

**Review by DraganDjuric94 — APPROVED** [review]

File: .bravo/config.yaml (hunk 1 above)
Comment (pajaxsg) [reply .bravo/config.yaml]: Done.

**Review by AmerSelimovic — APPROVED** [review]

**Review by DanijelaPet — APPROVED** [review]

File: .bravo/config.yaml [hunk 2]
```diff
@@ -12,28 +12,16 @@ pipeline:
...
-        chartPath: charts/eventrouter
-        lint: true
 
     "Build Docker Image":
       imageBuild: &imageBuild
         push: false
         name: platform/eventrouter
         tags:
-          - 0.3.0.pes1.0
+          - 0.3.0.pes2.0
```
Comment (dakamarko) [inline .bravo/config.yaml:21]: This should be 0.3.0.pes2

Comment (pajaxsg) [reply .bravo/config.yaml]: Done.

**Review by dakamarko — APPROVED** [review]

**Pull request merged**